cache/lru_cache.cc

```diff
@@ -596,15 +596,12 @@ void LRUCacheShard::Erase(const Slice& key, uint32_t hash) {
 
 bool LRUCacheShard::IsReady(Cache::Handle* handle) {
   LRUHandle* e = reinterpret_cast<LRUHandle*>(handle);
+  MutexLock l(&mutex_);
```

Comment:
Maybe add a test to verify that the mutex here does not case the performance regression?

Reply:
This function is no longer called from anywhere, but just there for completeness sake.